library/globals.inc.php

```diff
@@ -431,16 +431,9 @@
       ),
        '1',
       xl('Applies to Vitals form')
-    )
-      ,
-    'disable_deprecated_metrics_form' => array(
-      xl('Disable Old Metric Vitals Form'),
-       'bool',                          // data type
-       '1',                             // default = true
```

Comment:
Disabling this should remove all code attached to this a simple `grep -irn "disable_deprecated_metrics_form" .` will do the trick if you are on linux based OS
```
 muarachmann@Muas-MacBook-Pro  /Applications/MAMP/htdocs/lh-ehr   master ● ?  grep -irn "disable_deprecated_metrics_form" .
./interface/forms_admin/forms_admin.php:184:            if (($fname == "vitalsM") && ($GLOBALS['disable_deprecated_metrics_form'])) continue;
./library/globals.inc.php:436:    'disable_deprecated_metrics_form' => array(
./sql/nhanes/libreehr_nhanes.sql:103469:('disable_deprecated_metrics_form',	0,	'1'),
./sql/nhanes/libreehr_nhanes.sql:1610897:(813615,	'2018-04-30 17:56:15',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1610898:(813616,	'2018-04-30 17:56:15',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1611568:(814286,	'2018-04-30 18:04:49',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1611569:(814287,	'2018-04-30 18:04:49',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1612246:(814963,	'2018-04-30 20:11:58',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1612247:(814964,	'2018-04-30 20:11:58',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1614556:(817273,	'2018-06-26 15:01:21',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1614557:(817274,	'2018-06-26 15:01:21',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1615230:(817947,	'2018-06-26 15:03:08',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1615231:(817948,	'2018-06-26 15:03:08',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1615904:(818621,	'2018-06-26 15:04:49',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1615905:(818622,	'2018-06-26 15:04:49',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1616578:(819295,	'2018-06-26 15:06:28',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1616579:(819296,	'2018-06-26 15:06:29',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1617253:(819969,	'2018-06-26 15:08:09',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1617254:(819970,	'2018-06-26 15:08:09',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1617927:(820643,	'2018-06-26 15:09:50',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1617928:(820644,	'2018-06-26 15:09:50',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1618601:(821317,	'2018-06-26 15:11:31',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1618602:(821318,	'2018-06-26 15:11:31',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1619275:(821991,	'2018-06-26 15:13:15',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1619276:(821992,	'2018-06-26 15:13:16',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1619949:(822665,	'2018-06-26 15:15:19',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1619950:(822666,	'2018-06-26 15:15:19',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1620623:(823339,	'2018-06-26 15:17:22',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1620624:(823340,	'2018-06-26 15:17:22',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1621388:(824104,	'2018-07-27 13:51:05',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1621389:(824105,	'2018-07-27 13:51:05',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1623349:(826064,	'2018-09-30 05:29:08',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1623350:(826065,	'2018-09-30 05:29:09',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1624025:(826740,	'2018-09-30 06:51:25',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1624026:(826741,	'2018-09-30 06:51:25',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1624703:(827418,	'2018-09-30 07:43:27',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1624704:(827419,	'2018-09-30 07:43:27',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1625377:(828092,	'2018-09-30 07:45:14',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1625378:(828093,	'2018-09-30 07:45:14',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1626051:(828766,	'2018-09-30 07:47:01',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1626052:(828767,	'2018-09-30 07:47:01',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1627473:(830187,	'2018-10-18 09:45:22',	'other-delete',	'admin',	'Default',	'DELETE FROM `globals` WHERE gl_name = ? (\'disable_deprecated_metrics_form\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
./sql/nhanes/libreehr_nhanes.sql:1627474:(830188,	'2018-10-18 09:45:22',	'other-insert',	'admin',	'Default',	'INSERT INTO `globals` ( gl_name, gl_index, gl_value ) VALUES ( ?, ?, ? ) (\'disable_deprecated_metrics_form\',\'0\',\'1\')',	NULL,	0,	1,	'',	'',	'LibreEHR',	NULL,	NULL),
```